nixos/default.nix

```diff
@@ -45,6 +46,11 @@ in {
         <option>users.users.‹name?›.packages</option> option.
       '';
 
+      useGlobalPkgs = mkEnableOption ''
+        using <literal>pkgs</literal> from the NixOS modules in Home Manager.
```

Comment:
```suggestion
        use <literal>pkgs</literal> from the NixOS modules in Home Manager.
```

Reply:
`mkEnableOption` prefixes the description with 'Whether to enable'. The definition of
`useUserPackages` uses a similar style of accomodating to that prefix.

Reply:
Ah, I missed that. Thanks for the clarification!

Reply:
I think this is fine as-is. But in general, a useful idiom when the default description format of `mkEnableOption` is insufficient is to use something like
```nix
mkEnableOption "" // {
  description = ''
    A custom description goes here.
  '';
}
```